### DISCUSSION TITLE
more sizes /  top allignment

### DISCUSSION THREAD

**Discussion by BannukDE** [mulaRahul/keyviz#14] [Ideas]
Hello,

firstly thanks for the tool, really nice work.
Just the feature requests:
- I would like to have more sizes. 32px is bigger then my task bar.
- Alligning it at the top of the screen would be really awesome as well.

**Comment by mulaRahul** [comment]
Sounds good. I'll add more size options less than 32px in the next release. As for the alignment options other than the bottom, will try to add that in the subsequent releases.

**Comment by mulaRahul** [comment]
## 🪄 Planned Improvements
- Replace keycap size **dropdown** with a **slider**/**_spinbox_** with **16px** as lower limit and **64px** as upper limit.
- Add more visualization alignment options - Top Left, Top Center, Top Right, Center Left, ~~Center,~~ and Center Right.
- Expose **border transparency**.
- Expose **border radius**.

**Reply by andreincu** [reply]
Did you take into consideration to offer the ability to enter custom values?

**Reply by mulaRahul** [reply]
Yes, as mentioned dropdowns will be replaced with spinbox in which you can enter custom values. 

_Something like this_ - 
![spinbox](https://openclipart.org/image/100px/177397)

**Comment by mulaRahul** [comment]
Added with [v2.0.0a](https://github.com/mulaRahul/keyviz/releases/tag/v2.0.0a).
